platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -166,11 +164,13 @@ protected boolean doLogin() throws LoginException {
         // This is to prevent someone from logging into Karaf as any user without providing a
         // valid password (because if authentication = none, the password could be any
         // value - it is ignored).
-        if ("none".equals(authentication) && (tmpPassword != null)) {
+        // Username is not checked in this conditional because a null username immediately exits
+        // this method.
+        if ("none".equalsIgnoreCase(getBindMethod()) && (tmpPassword != null)) {
             LOGGER.debug(
                     "Changing from authentication = none to simple since user or password was specified.");
             // default to simple so that the provided user/password will get checked
-            authentication = "simple";
```

Comment:
Unit testing caught an error in the code. The authentication variable was being set, but not used to decide what auth method should be used. Changed to setting the binding method property.